Python/Product/PythonTools/PythonTools/Repl/PythonInteractiveEvaluator.cs

```diff
@@ -661,7 +661,9 @@ partial class PythonInteractiveEvaluator :
         }
 
         internal Task InvokeAsync(Action action) {
-            return ((System.Windows.UIElement)_window.GetTextViewHost()).Dispatcher.InvokeAsync(action).Task;
+            // TOOD: Verify this
```

Comment:
This is the thing I need to test. Does it look like it would work? There's no GetTextViewHost() method on the shared implementation.